packages/docs/docs/getting-started/logging.mdx

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 12
+slug: /logging
+---
+
+# Logging
+
+One method of debugging your code or animation flow is using logging messages.
+For this, motion-canvas has its own build-in way to log messages.
+
+To get a reference to the Logger in motion-canvas you can use the useLogger
```

Comment:
Do you want to add ticks around useLogger (i.e. replace useLogger with `useLogger`)?